### ISSUE TITLE
Procuring license for commerical application of YOLO - NAs

### ISSUE THREAD

**Issue by agnanas** [Deci-AI/super-gradients#2034]
### 💡 Your Question

Hi,

I am impressed with the capabilities of the yolo-nas models and would like to see how I would go about purchasing the licenses for the pre-trained weights. However, given the recent developments in Deci, I cannot find the pages linked to sales or licensing. Could you please let me know whom to contact for this or point me in the direction where I can find more relevant information?

Thanks in advance!
